app/controllers/compute_resources_controller.rb

```diff
@@ -70,6 +75,11 @@ def update
     else
       process_error
     end
+  rescue StandardError => e
```

Comment:
Why do we catch `StandardError` here and `OVIRT::OvirtException` above? I expect that we should behave the same in both cases.
And if we want to catch a more generic error class, I would go with `Fog::Errors::Error` perhaps which fog-ovirt should inherit for its errors?

Reply:
Okay, following our conversation f2f, I'ved also added a "fall-back" so both `Fog::Error::Error` and `OVIRT::OvirtException` will be captured.